### PR TITLE
LPS-203496 | Add a test to get the fields annotated with GraphQLTypeExtension in GraphQL

### PR DESCRIPTION
**Pull request by Hijky**
https://liferay.atlassian.net/browse/LPS-203250

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

File: modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/tests/HeadlessDiscovery.testcase [hunk 1]
```diff
@@ -117,6 +117,36 @@ definition {
...
+		property portal.acceptance = "true";
+
+		task ("Given a basic web content is created") {
+			HeadlessWebcontent.addBasicWebContent(
+				content = "Web Content Content",
+				groupName = "Guest",
+				title = "Web Content Title");
+		}
+
+		task ("Given navigate to headless delivery endpoint") {
```
Comment (yanan) [inline modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/tests/HeadlessDiscovery.testcase:131]: And Given?

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
The failures are about API builder response messages not related to PR.

**Review by JoyceWang08211 — APPROVED** [review]
LGTM

**Comment by Hijky** [comment]
ci:forward

**Pull request closed without merging**